inc/Validation/Validator.php

```diff
@@ -25,6 +25,10 @@ public function validate( array $data ): bool|WP_Error {
 	}
 
 	private function validate_schema( array $schema, $data ): bool|WP_Error {
```

Comment:
Should we use an off the shelf library for this type of thing?

Reply:
I looked a few things:
- Straight reuse of WordPress REST validation/sanitization - https://github.com/Automattic/remote-data-blocks/pull/83
- Concocting something directly JSON Schema 4-based.
- `Respect/Validation` - https://github.com/Respect/Validation
- Symfony Validation - https://symfony.com/doc/current/validation.html
- And glanced at a few others like Laravel validation, `rakit/validation`, and `vlucas/valitron`.

If you have something in mind, I'm certainly open to it. I just ended up here--a WP REST-like which is JSON Schema 4ish itself--after weighing various pros/cons.

Reply:
I'm ok with this, but if we find ourselves needing more and more from it, it might be worth more exploration.